README.md

```diff
@@ -75,8 +78,32 @@ Configure a remote node with a Conjur identity:
       conjur_account: 'myorg',
       conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}",
       conjur_host_name: "{{inventory_hostname}}"
+    - summon
```

Comment:
I think it would be better if we include the summon install tasks under the role, otherwise we're shipping 2 roles in this repository and that's confusing. The other path would be to create a repo/role just for summon on its own, but to me that seems overkill at this point.

Reply:
I had left it out to keep it flexible, but offering a single role to install summon makes a ton of sense.